api/instance/v1/server_utils.go

```diff
@@ -4,25 +4,20 @@ import (
 	"time"
 
 	"github.com/scaleway/scaleway-sdk-go/internal/async"
+	"github.com/scaleway/scaleway-sdk-go/internal/errors"
 	"github.com/scaleway/scaleway-sdk-go/utils"
 )
 
-// WaitForServerRequest is used by WaitForServer method
-type WaitForServerRequest struct {
+// waitForServerRequest is used by waitForServer method
+type waitForServerRequest struct {
 	ServerID string
 	Zone     utils.Zone
-
-	// Timeout: maximum time to wait before (default: 5 minutes)
-	Timeout time.Duration
+	Timeout  time.Duration
 }
 
-// WaitForServer wait for the server to be in a "terminal state" before returning.
+// waitForServer wait for the server to be in a "terminal state" before returning.
 // This function can be used to wait for a server to be started for example.
-func (s *API) WaitForServer(req *WaitForServerRequest) (*Server, error) {
-
-	if req.Timeout == 0 {
-		req.Timeout = 5 * time.Minute
-	}
+func (s *API) waitForServer(req *waitForServerRequest) (*Server, error) {
```

Comment:
now that this is private it should return a SdkError ?

Reply:
done